docs/diagrams/evidence-request-creation-sequence-flow.iuml

```diff
@@ -0,0 +1,18 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.2.0/C4_Deployment.puml
+!include https://raw.githubusercontent.com/LBHackney-IT/cv-19-res-support-v3/development/docs/diagrams/c4_shared.puml
+
+title Evidence request creation sequence flow diagram
+
+Officer -> Next.JS: Search for resident
+Next.JS -> EvidenceAPI: GET /residents/:residentId
+Next.JS <- EvidenceAPI: 200 Resident found
+Officer <- Next.JS: Resident found
```

Comment:
I know on Miro we have it labelled as _Next.JS_ but maybe here, and on the other diagrams, we could name it something more descriptive. Right now it's named after the technology used, while Evidence and Documents API have their actual names. Maybe something like _Frontend_ or _Evidence UI_

Reply:
Ok, I'll change it to `Frontend`